package.json

```diff
@@ -17,6 +17,7 @@
     "postinstall": "yarn typechain && yarn sdk"
   },
   "dependencies": {
+    "@eth-optimism/sdk": "^1.1.7",
```

Comment:
The yarn.lock file has some troubles. They have +/- changes for a single package.

Reply:
The yarn.lock file has some troubles. It has "+/-" changes for a single package.